### PR TITLE
DSP-149 : ec2VSCode : add a launcher for code server version 1.71.0

### PR DESCRIPTION
**Pull request by pawanrlelab**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710LaunchSSM.yaml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+        Type: String
+        Selector: '$.RecordDetail.ProvisionedProductId'
+      - Name: RecordId
+        Type: String
+        Selector: '$.RecordDetail.RecordId'
+    description: Launch Vscode
+    nextStep: WaitForProvisioning
+  - name: WaitForProvisioning
+    action: 'aws:waitForAwsResourceProperty'
+    timeoutSeconds: 600
```
Comment (manikandan-thangavelu-rl) [inline solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710LaunchSSM.yaml]: Can you change this to 1200, Same change for termination ssm also. Make sure you do this for both versions

File: solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710TerminateSSM.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      ProvisionedProductId: '{{ ProvisionedProductId }}'
+      TerminateToken: '{{ TerminateToken }}'
+    outputs:
+      - Name: RecordId
+        Type: String
+        Selector: '$.RecordDetail.RecordId'
+    nextStep: WaitForTerminating
+  - name: WaitForTerminating
+    action: 'aws:waitForAwsResourceProperty'
+    timeoutSeconds: 600
```
Comment (manikandan-thangavelu-rl) [inline solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710TerminateSSM.yaml]: Added a comment above

**Review by manikandan-thangavelu-rl — CHANGES_REQUESTED** [review]

File: solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710TerminateSSM.yaml (hunk 2 above)
Comment (pawanrlelab) [reply solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710TerminateSSM.yaml]: made the suggested changes in commit c3ac4a835b3b6886d84d5c843038e0f81376215b

File: solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710LaunchSSM.yaml (hunk 1 above)
Comment (pawanrlelab) [reply solutions/swb-reference/src/environment/ec2VSCode1710/ec2VSCode1710LaunchSSM.yaml]: made the suggested changes in commit c3ac4a835b3b6886d84d5c843038e0f81376215b

**Pull request merged**